lib/Echidna/Campaign.hs

```diff
@@ -272,7 +346,8 @@ randseq ql o w = do
       -- Randomly generate new random transactions
       gtxs <- replicateM ql (evalStateT (genTxM o) (w, ca ^. genDict))
       -- Select a random mutator
-      mut <- seqMutators cs
+      cmut <- seqMutators cs
+      let mut = getCorpusMutation cmut
```

Comment:
`let mut = seqMutators cs >>= getCorpusMutation`

Reply:
That won't work. :(